.gitignore

```diff
@@ -9,5 +9,10 @@
 .Rproj.user
 README.html
 .RData
-pct-data/
+/regions_www/tabs/download_params_sha_current.rds
```

Comment:
This looks like an absolute file path to me. I suggest this: 

```
regions_www/tabs/download_params_sha_current.rds
```

Right? @nikolai-b

Reply:
https://git-scm.com/docs/gitignore

In this case both are valid forms, maybe there is a difference more subtle that I'm unaware of.